modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,254 @@
+    affinity:
+      nodeAffinity: <1>
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - values:
+                    - ip-10-0-163-94.us-west-2.compute.internal
+                  key: kubernetes.io/hostname
+                  operator: In
+ ...
+----
+<1> A node affinity that requires the Operator's pod to be scheduled on a node named `ip-10-0-163-94.us-west-2.compute.internal`.
+
+.Node affinity example that places the Operator pod on a node with a specific architecture
+[source,terminal]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  creationTimestamp: '2022-12-21T14:54:47Z'
+  generation: 6
+  labels:
+    operators.coreos.com/openshift-custom-metrics-autoscaler-operator.openshift-keda: ''
+  name: openshift-custom-metrics-autoscaler-operator
+  namespace: openshift-keda
+  resourceVersion: '77887'
+  uid: 9b38d0eb-9c57-4cb6-9905-1c65c21f08ef
+spec:
+  name: my-package
+  source: my-operators
+  sourceNamespace: operator-registries
+  config:
+    affinity:
+      nodeAffinity: <1>
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: kubernetes.io/arch
+              operator: In
+              values:
+              - arm64
+            - key: kubernetes.io/os
+              operator: In
+              values:
+                - linux
+----
+<1> A node affinity that requires the Operator's pod to be scheduled on a node with the `kubernetes.io/arch=arm64` and `kubernetes.io/os=linux` labels.
+endif::pod[]
+
+ifdef::pod[]
+The following example shows how to use pod anti-affinity to prevent the installation the Custom Metrics Autoscaler Operator from any node that has pods with a specific label.
+endif::pod[]
+
+ifndef::node[]
+.Pod anti-affinity example that prevents the Operator pod from one or more specific nodes
+[source,terminal]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  creationTimestamp: '2022-12-21T14:54:47Z'
+  generation: 6
+  labels:
+    operators.coreos.com/openshift-custom-metrics-autoscaler-operator.openshift-keda: ''
+  name: openshift-custom-metrics-autoscaler-operator
+  namespace: openshift-keda
+  resourceVersion: '77887'
+  uid: 9b38d0eb-9c57-4cb6-9905-1c65c21f08ef
+spec:
+  name: my-package
+  source: my-operators
+  sourceNamespace: operator-registries
+  config:
+    affinity:
+      podAntiAffinity: <1>
+        requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+            - key: cpu
+              operator: In
+              values:
+              - high
+          topologyKey: kubernetes.io/hostname
+ ...
+----
+<1> A pod anti-affinity that prevents the Operator's pod from being scheduled on a node that has pods with the `cpu=high` label.
+endif::node[]
+
+.Procedure
+
+To control the placement of an Operator pod:
```

Comment:
```suggestion
To control the placement of an Operator pod, complete the following steps:
```
See ISG on [Introducing the procedure](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#introducing-the-procedure)